GSoC-2025/ethology_support_for_any_point_tracking_vietluong.md

```diff
@@ -0,0 +1,148 @@
+|      | - Fix any remaining frontend/backend bugs                                          |       |
+| 10   | **Detailed documentation and user guide preparation**                              | 30    |
+|      | - Complete comprehensive backend/frontend documentation                            |       |
+|      | - Write detailed user guides/tutorials                                             |       |
+|      | - Prepare usage examples                                                           |       |
+| 11   | **Code freeze, final testing, and submission preparation**                         | 30    |
+|      | - Conduct final comprehensive tests                                                |       |
+|      | - Address any critical bugs or performance issues                                  |       |
+|      | - Begin preparation of final submission materials                                  |       |
+| 12   | **Final feedback incorporation & project submission**                              | 30    |
+|      | - Incorporate mentor and community feedback                                        |       |
+|      | - Finalize and polish all documentation                                            |       |
+|      | - Submit final deliverables                                                        |       |
+
+
+**Total Project Duration:** 12 weeks × 30 hours/week = **~360 hours**
+
+**Note:** No vacations or major commitments planned; weekly hours and tasks may adjust slightly based on mentor feedback and project progress.   
+
+- **Communication plan**
+
+- **Frequency:** Weekly stand-up meetings for updates and progress reviews; additional brief daily or bi-weekly check-ins via chat as needed.
+- **Channels:** Regular video calls (Zoom/Google Meet) for weekly discussions; continuous communication and quick queries via Zulip or GitHub issues.
+## Personal statement
+
+_Length: max 0.75 page_
+
+- **Past experienc.** 
+
+I have a robust background in programming, open-source development, and research, highlighted by my ongoing roles as a Research Assistant at Yin Li’s Lab and Vikas Singh’s Lab at the University of Wisconsin-Madison. At Yin Li’s Lab, I designed and implemented algorithms to generate precise region prompts for the MedSAM model, enhancing localization accuracy in medical X-ray images. Additionally, I successfully applied CLIPSelf to improve dense feature representation, significantly boosting the model's accuracy.
+
+At Vikas Singh’s Lab, I focused on hyperparameter tuning and model optimization for the SimpleTM model, executing extensive experiments on datasets such as PEMS and M4. Utilizing JAX and Triton frameworks, I optimized model efficiency, improving speed per iteration by sixfold. My published work at the International Conference on Representation Learning (ICLR 2025) reflects these efforts, establishing new benchmarks in multivariate time-series forecasting.
```

Comment:
Could you add a bit more on what was the problem/task that the SimpleTM model was trying to solve? 

Did your work involve the definition of new benchmarks for multivariate time-series forecasting? If so, what did these benchmark check?